csharp/src/SeedLang.Block/Parser.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 The Aha001 Team.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using SeedLang.X;
+
+namespace SeedLang.Block {
+  // A parser class which is responsible to convert a block program to an AST tree, an inline
+  // expression text to a list of blocks, and an inline expression text to an AST tree.
```

Comment:
A clearer expression for the comments:

```
// Converts a block program to an AST tree, an inline expression text to a list of blocks,
// or an inline expression text to an AST tree.
```

(Optional) Is it good to clarify in the comments that SeedLang.Block.Parser is different from Seedlang.X.BlockParser? The two classes look a bit confusing to newcomers. It'd probably be good to comment on the relationship between the two.

Reply:
I adjusted this comment but still keep the subject "The parser", because this is a comment for a class instead of a method. A statement to mention the diffenece with SeedLang.X.BlockParser is also added.